api/app/controllers/mno_enterprise/jpi/v1/app_answers_controller.rb

```diff
@@ -0,0 +1,21 @@
+module MnoEnterprise
+  class Jpi::V1::AppAnswersController < Jpi::V1::AppReviewsController
+
+    # GET /mnoe/jpi/v1/marketplace/:id/app_comments
+    def index
```

Comment:
I think I would prefer that you introduce a scope_index or scope method, that would be overriden by the children classes, (that would do the where) and would be called by the index mother method.